Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -3542,9 +3551,6 @@ + (OWSSignalServiceProtosEnvelope *)createEnvelopeForThread:(TSThread *)thread
                                                           withTransaction:transaction]];
         [result addObject:[TSErrorMessage invalidVersionWithEnvelope:[self createEnvelopeForThread:thread]
                                                      withTransaction:transaction]];
-        [result addObject:[TSInvalidIdentityKeyReceivingErrorMessage
-                              untrustedKeyWithEnvelope:[self createEnvelopeForThread:thread]
-                                       withTransaction:transaction]];
```

Comment:
IMO it'd be nice to keep this so we can use this to ensure that we don't break users who have these old messages.

Reply:
Good point - I've added this back.